src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/smoke/person/SentencesSmokeTest.kt

```diff
@@ -36,11 +36,27 @@ class SentencesSmokeTest : DescribeSpec(
           "data": [
             {
               "dateOfSentencing": "2019-08-24",
-              "isActive": null
+              "isActive": null,
+              "terms": [
+                {
+                  "years": 1,
+                  "months": 2,
+                  "weeks": 3,
+                  "days": 4
+                }
+              ]
             },
             {
               "dateOfSentencing": "2019-08-24",
-              "isActive": true
+              "isActive": true,
+              "terms": [
+                {
+                  "years": null,
+                  "months": null,
+                  "weeks": null,
+                  "days": null
```

Comment:
Whilst this is true for right now, if there is no sentence term information why are we returning a bunch of nulls?
We are technically CREATING data when there isn't any by declaring it null. Instead this should match the upstream functionality and return an empty list when there are no terms.